gradient/api_sdk/models/notebook.py

```diff
@@ -8,10 +8,38 @@
 class Notebook(object):
     id = attr.ib(type=str, default=None)
     vm_type_id = attr.ib(type=int, default=None)
+    vm_type_label = attr.ib(type=str, default=None)
     container_id = attr.ib(type=int, default=None)
     container_name = attr.ib(type=str, default=None)
     name = attr.ib(type=str, default=None)
-    cluster_id = attr.ib(type=int, default=None)
+    cluster_id = attr.ib(type=str, default=None)
+    registry_username = attr.ib(type=str, default=None)
+    registry_password = attr.ib(type=str, default=None)
+    default_entrypoint = attr.ib(type=str, default=None)
+    container_user = attr.ib(type=str, default=None)
+    shutdown_timeout = attr.ib(type=int, default=None)
+    is_preemptible = attr.ib(type=bool, default=None)
+    project_id = attr.ib(type=bool, default=None)
+    state = attr.ib(type=bool, default=None)
+    vm_type = attr.ib(type=bool, default=None)
+    fqdn = attr.ib(type=bool, default=None)
+    namespace = attr.ib(type=str, default=None)
+    tags = attr.ib(type=list, factory=list)
+
+    @property
+    def url(self):
+        url = concatenate_urls(config.WEB_URL, "/console/{}/notebook/{}".format(self.namespace, self.project_id))
+        return url
+# Is this the right way to do this?
```

Comment:
Not exactly :) Models are supposed to represent resource, which is Notebook in this case. I cannot imagine how NotebookStart could be a valid entity, but if it is, something might be messed up. I need to look into PS_API endpoint and get back to this, but it must be possible to solve it other way.

Btw: please try to follow python's pep8 style guidelines and use two lines between classes (if you use PyCharm, it should highlight this for you, also you can use Cmd/Ctrl+Shift+L shortcut to auto-align your code)